src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -785,7 +785,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Get
                 If _lazyAllSyntaxTrees.IsDefault Then
                     Dim builder = ArrayBuilder(Of SyntaxTree).GetInstance()
-                    builder.AddRange(_rootNamespaces.Keys)
```

Comment:
I think we should rename _rootNamespaces. Its name is confusing because it doesn't contain information about embedded trees, but one might assume it does.